plugins/services/src/js/components/forms/ContainerServiceFormSection.js

```diff
@@ -98,7 +98,9 @@ class ContainerServiceFormSection extends Component {
     const inputFiled = (
       <FieldInput
         key="gpus-input"
+        min="0.001"
```

Comment:
It's GPUs and we shouldn't have a default other then `0` as  gpu support is still experimental.